src/Agent.Listener/CommandSettings.cs

```diff
@@ -163,6 +163,31 @@ public bool GetRunAsService()
                 defaultValue: false);
         }
 
+        public bool GetEnableAutoLogon()
+        {
+            return TestFlagOrPrompt(
+                name: Constants.Agent.CommandLine.Flags.EnableAutoLogon,
+                description: StringUtil.Loc("EnableAutoLogon"),
+                defaultValue: true);
```

Comment:
i would think `False` should be the default value, otherwise we will break existing unattend config script customer had.